using_images/s2i_images/dot_net_core.adoc

```diff
@@ -66,8 +66,73 @@ link:https://github.com/redhat-developer/s2i-dotnetcore/blob/master/dotnet_image
 [[dot-net-core-configuration]]
 == Configuration
 
-The .NET Core image currently does not support any environment variables. No
-configuration is necessary.
+The .NET Core images support a number of environment variables, which you can
+set to control the build behavior of your .NET Core application.
+
+[NOTE]
+====
+You must set environment variables that control build behavior in the s2i build
+configuration or in the *_.s2i/environment_* file in to make them available to
+the build steps.
+====
+
+.NET Core Environment Variables
+[cols="4a,6a,6a",options="header"]
+|===
+
+|Variable Name |Description |Default
+
+|`DOTNET_STARTUP_PROJECT`
+|Used to select the project to run. This must be the folder containing *_project.json_*.
+|Current working directory.
```

Comment:
It's made more clear by adding "in the source repository" in the description.

Current directory may mean something else to the user:
```
~/repos/new-webapp/src (master)$ oc new-app ...
```